Cargo.lock

```diff
@@ -811,6 +811,7 @@ dependencies = [
  "rust-crypto 0.2.36 (registry+https://github.com/rust-lang/crates.io-index)",
  "rustc-hex 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "rustc-serialize 0.3.24 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I just noticed, you most forgot to remove `rust-serialize` from `ethcore/Cargo.toml` and and `util/network/Cargo.toml`

Reply:
My bad, looks like I searched using underscore instead of dash.